packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -22,7 +22,12 @@ export class LoginPage {
 	 */
 	async visit( { path }: { path: string } = { path: '' } ): Promise< Response | null > {
 		const targetUrl = path ? `log-in/${ path }` : 'log-in';
-		return await this.page.goto( getCalypsoURL( targetUrl ), { waitUntil: 'networkidle' } );
+		// We are getting a pending status for https://wordpress.com/cspreport intermittently
+		// which causes the login to hang on networkidle when running the tests locally.
+		// This continues the route request.
+		// See https://github.com/Automattic/wp-calypso/issues/69294
+		await this.page.route( '**/cspreport', ( route ) => route.continue() );
```

Comment:
**question:non-blocking**: I'm curious about the `continue` here.

It does not appear that the request to `cspreport` is being overridden, so would this not lead to the same intermittent timeout?

I'm wondering if a `fulfill` with HTTP 200 would be more possibly more reliable.